uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveRequestStatusMatrixTest.php

```diff
@@ -185,4 +258,18 @@ public function allNonPossibleStatusTransitionForLeaveApprover() {
       ['', $leaveRequestStatuses['cancelled']],
     ];
   }
+
+  /**
+   * Return possible status transitions that is only exclusive to the Manager or Admin
+   *
+   * @return array
+   */
+  public function getManagerExclusivePossibleStatusTransitions() {
```

Comment:
While it's fine like this, it might be worth looking into [data providers](https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers) for this kind of test in the future.